### ISSUE TITLE
TSMP build log -- Git info

### ISSUE THREAD

**Issue by niklaswr** [HPSCTerrSys/TSMP#145]
## TSMP build log -- Git info
I could not find the commit when this was introduced, but TSMP is logging some information of the different component repos used.   
First of all, this is nice, but I would suggest to add the remote URL as well.  

What is logged now (example):
```
[...]
Git (clm3_5):                                                                   
/p/scratch/cesmtst/wagner6/ERA5Climat_EUR11_ECMWF-ERA5_analysis_FZJ-IBG3_v02/src/TSMP/clm3_5/.git
master                                                                          
801b530 
[...]
```
What I want to be logged:
```
Git (clm3_5):                                                                   
/p/scratch/cesmtst/wagner6/ERA5Climat_EUR11_ECMWF-ERA5_analysis_FZJ-IBG3_v02/src/TSMP/clm3_5/.git
--> https://icg4geo.icg.kfa-juelich.de/ModelSystems/tsmp_src/clm3.5_fresh.git
master                                                                          
801b530 
[...]
```

I think this could be useful, as the absolute path may change, but the URL (usually) not.

**Comment by s-poll** [comment]
Fully agree with the suggestion.

I recognized that the git info currently leads to failure for CLM3.5 build and setup in combination with ICON. It is searching for `clm-icon` directory instead of `clm`, which results in an aborption of the build/setup.

**Comment by jjokella** [comment]
> What I want to be logged:
> 
> ```
> Git (clm3_5):                                                                   
> /p/scratch/cesmtst/wagner6/ERA5Climat_EUR11_ECMWF-ERA5_analysis_FZJ-IBG3_v02/src/TSMP/clm3_5/.git
> --> https://icg4geo.icg.kfa-juelich.de/ModelSystems/tsmp_src/clm3.5_fresh.git
> master                                                                          
> 801b530 
> [...]
> ```

I have implemented `git remote get-url origin` in #149 that could potentially solve this issue, there are also some downsides that I state in the PR.

**Comment by jjokella** [comment]
> I recognized that the git info currently leads to failure for CLM3.5 build and setup in combination with ICON. It is searching for `clm-icon` directory instead of `clm`, which results in an abortion of the build/setup.

I implemented `getGitInfo` such that it uses the array `mList` that is filled with `modelVersion`:

https://github.com/HPSCTerrSys/TSMP/blob/a01f15ac4d0f0f1a72a80b495582b025724781a2/bldsva/supported_versions.ksh#L72-L94

In many places of TSMP, `mList` is used for directory, where the component model source code is located. So I think, the error for `clm` / `clm-icon` poses the larger question: Do we want `mList` to be the name of the component model directory or not.
